scripts/gh-actions/parse_versions.py

```diff
@@ -4,22 +4,23 @@
 
 # Parse the versions given as a comma-separated list and return a JSON array
 def parse_versions(input_versions):
-    # Default version string if the input is empty
-    if not input_versions:
-        input_versions = "1.8,1.9,latest"
-    else:
-        # Remove whitespace between entries
-        input_versions = input_versions.replace(" ", "")
+    # Remove double quotes
+    input_versions = input_versions.strip('"')
+    # Remove whitespace between entries
+    input_versions = input_versions.replace(" ", "")
     
     # Convert to JSON array
     json_array = json.dumps(input_versions.split(","))
     return json_array
 
 if __name__ == "__main__":
-    # Read the input of the Github Action from the environment variable
-    input_versions = os.getenv('INPUT_BUNDLE_VERSION', '')
+    if len(sys.argv) < 1:
+        raise Exception("No bundle versions given as input.")
```

Comment:
do you know what's the case where this exception will apply? I'm thinking that since the input `bundle_version` is required, then it will not be possible to run the workflow dispatch without passing the input. I even tried running it with `bundle_version` as an empty string `"` and it still picked up the defaults `1.8, 1.9, latest`.
Can you please confirm?

Reply:
That is correct, this `if` case exists for the scenario where the script is used in other actions, and not receiving any arguments should be treated as an error.